core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -27,15 +27,17 @@ object WorkflowSubmissionActor {
             executionServiceCluster: ExecutionServiceCluster,
             batchSize: Int,
             credential: Credential,
+            processInterval: FiniteDuration,
             pollInterval: FiniteDuration,
             maxActiveWorkflowsTotal: Int,
             maxActiveWorkflowsPerUser: Int,
             runtimeOptions: Option[JsValue]): Props = {
-    Props(new WorkflowSubmissionActor(dataSource, methodRepoDAO, googleServicesDAO, executionServiceCluster, batchSize, credential, pollInterval, maxActiveWorkflowsTotal, maxActiveWorkflowsPerUser, runtimeOptions))
+    Props(new WorkflowSubmissionActor(dataSource, methodRepoDAO, googleServicesDAO, executionServiceCluster, batchSize, credential, processInterval, pollInterval, maxActiveWorkflowsTotal, maxActiveWorkflowsPerUser, runtimeOptions))
   }
 
   sealed trait WorkflowSubmissionMessage
-  case object ScheduleNextWorkflowQuery extends WorkflowSubmissionMessage
```

Comment:
Since it's scheduling the next `LookForWorkflows` and may not necessarily process any workflows if it doesn't find any, some descriptor of that in the name is appropriate.  `-Query` may not be the best suffix but it's better than nothing.  Maybe `-Search` ?

`ProcessNextWorkflow` works because we know there are workflows to process.